src/plugins/projectexplorer/projectexplorer_dependencies.pri

```diff
@@ -2,6 +2,7 @@ QTC_PLUGIN_NAME = ProjectExplorer
 QTC_LIB_DEPENDS += \
     aggregation \
     extensionsystem \
+    sfdk \
```

Comment:
Really needed?

Reply:
Yes, without it dos't work

Reply:
Hm, are you able to tell why?

Reply:
We use members from sfdk libs and without it dos't work